### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by thednp**
First attempt to add Native Javascript for Bootstrap to jsdelivr CDN

### CODE REVIEW

File: files/bootstrap.native/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "dnp_theme"
+github = "https://github.com/thednp/bootstrap.native"
+homepage = "http://themeforest.net/user/dnp_theme"
+description = "If you are looking for Bootstrap without jQuery or vanilla Javascript for Bootstrap, this is the thing."
+mainfile = "dist/bootstrap-native.min.js"
```
Comment (jimaek) [inline files/bootstrap.native/info.ini]: Please remove dist/, the bot wont create that directory.

Comment (thednp) [inline files/bootstrap.native/info.ini]: Please tell me, should I also add files to root? or simply leave them in the dist folder?

Comment (jimaek) [inline files/bootstrap.native/info.ini]: You can leave them as is. Our bot will copy them from there but it wont create the directory, so you need to remove dist/ from info.ini

Comment (thednp) [inline files/bootstrap.native/info.ini]: Done.

**Comment by jimaek** [comment]
Its correct, but you are over our limit of 2 commits. Either squash your commits into 1 or create a new PR

**Comment by thednp** [comment]
I don't know how to squash them, I shoud close this one and create a new one.

**Pull request closed without merging**